eclipse-scout-core/src/form/fields/fields.ts

```diff
@@ -175,5 +176,27 @@ export const fields = {
   focusAndInsertText($input: JQuery<HTMLInputElement | HTMLTextAreaElement>, text: string) {
     $input[0].focus();
     $input.document(true).execCommand('insertText', false, text);
+  },
+
+  /**
+   * @returns true if the element supports the readonly attribute
+   */
+  supportsReadOnlyAttribute($element: JQuery<HTMLInputElement | HTMLTextAreaElement>) {
+    let $elem = $.ensure($element);
+    if (!$elem.length) {
+      return false;
+    }
+    if ($elem.is('textarea')) {
+      return true;
+    }
+    if (!$elem.is('input')) {
```

Comment:
Can this happen? The parameter type indicates that only input and text area elements can be passed. If this check should stay, maybe allow all jquery elements in the parameter?